### ISSUE TITLE
Error rendering vector line with very wide style

### ISSUE THREAD

**Issue by robinpaulson** [nextgis/QTiles#70]
I rendered the map I've been making today, attached are a sample of the screenshots it produced. They are of the same position, at zooms 20, 21 and 22 respectively.

As you can see, as I zoom in, the thick grey lines (which should have parallel lines) become more and more degraded. The errors lineup with the edge of the tiles. The style used to draw the vector line renders it as 5m wide, which I think is the issue - as the style overlaps on to a different tile from the origin line, so it fails to render. This error is consistent across the entire tile set.

I think this may be related to some of the other rendering bugs which are listed in other issues.

I may have explained this really badly, please ask for clarification if necessary.

![z20](https://cloud.githubusercontent.com/assets/1186107/17317946/b441c7f8-58d6-11e6-82d6-9148e27de8a3.png)
![z21](https://cloud.githubusercontent.com/assets/1186107/17317948/b483ea70-58d6-11e6-8347-29acb6b9a219.png)
![z22](https://cloud.githubusercontent.com/assets/1186107/17317947/b463bff2-58d6-11e6-9033-c4f71dd9c1ae.png)

Cheers

**Comment by robinpaulson** [comment]
I investigated further, it appears the rendering errors only appear when the line style is wider than 1.5m. The grey line above is 4m wide

**Comment by robinpaulson** [comment]
No, scrub that. It turns out the errors happen with any style if you take the zoom far enough.

**Comment by olcayebcin** [comment]
Is there any idea how to overcome this problem?
